chrome/content/zotero/xpcom/translation/translate_item.js

```diff
@@ -786,11 +786,11 @@ Zotero.Translate.ItemGetter.prototype = {
 		this.numItems = this._itemsLeft.length;
 	},
 	
-	"setAll":function(getChildCollections) {
-		this._itemsLeft = Zotero.Items.getAll(true);
+	"setAll":function(libraryID, getChildCollections) {
+		this._itemsLeft = Zotero.Items.getAll(true, libraryID);
 		
 		if(getChildCollections) {
-			this._collectionsLeft = Zotero.getCollections();
+			this._collectionsLeft = Zotero.getCollections(null, false, libraryID);
```

Comment:
The second parameter here should be `true` (i.e. recursively) as is the case for `setCollection` above. Looks like this was a bug.